build/build.xml

```diff
@@ -101,6 +108,17 @@ External Dependencies:
 		<property name="src.isInit" value="true" />
 	</target>
 
+	<!-- create script to run built commandbox using sources (for core cfml development) -->

```

Comment:
Don't know.  It's something @denuno added that I think just builds the CFML bits.